src/swap/SwapScreen.tsx

```diff
@@ -265,9 +266,7 @@ export function SwapScreenSection({ showDrawerTopNav }: { showDrawerTopNav: bool
       }))
     }
 
-    if (fieldType === Field.FROM) {
-      setShowMaxSwapAmountWarning(false)
-    }
```

Comment:
we should really clear this warning when the "to" amount changes too, since that would also change the "from" amount